packages/card/enable.js

```diff
@@ -0,0 +1,3 @@
+window.Vaadin ||= {};
+window.Vaadin.featureFlags ||= {};
+window.Vaadin.featureFlags.cardComponent = true;
```

Comment:
I think we should rather not have separate `enable.js` as this file will have to be added to dev bundles.
Instead I'd recommend using the approach I [prototyped](https://github.com/vaadin/web-components/compare/proto/experimental-component) (I can create a separate PR for it).